odl/operator/operator.py

```diff
@@ -31,7 +31,7 @@
 from odl.set import LinearSpace, LinearSpaceElement, Set, Field
 
 
-__all__ = ('Operator', 'OperatorComp', 'OperatorSum',
+__all__ = ('Operator', 'OperatorComp', 'OperatorSum', 'OperatorVectorSum',
```

Comment:
Do we need all those expression types in the top level namespace? In my view, they clutter substantially, and most users don't want to see them since they never use them directly.